rslib/src/notetype/fields.rs

```diff
@@ -35,18 +35,22 @@ impl NoteField {
         }
     }
 
-    pub(crate) fn fix_name(&mut self) {
-        // remove special characters
+    pub(crate) fn fix_name(&mut self) -> bool {
+        // Return whether the name is valid. Remove special characters if it leads to a valid name.
```

Comment:
To make this a docstring, it should start with three slashes (///), and appear above the function name.